src/main/java/org/spongepowered/common/mixin/api/mcp/entity/boss/dragon/EnderDragonEntityMixin_API.java

```diff
@@ -79,4 +81,16 @@ public ServerBossBar getBossBar() {
     public DragonPhaseManager getPhaseManager() {
         return (DragonPhaseManager) this.phaseManager;
     }
+
+    @Override
+    protected Set<Value.Immutable<?>> api$getVanillaValues() {
+        final Set<Value.Immutable<?>> vanillaValues = super.api$getVanillaValues();
+
+        if (this.closestEnderCrystal != null) {
```

Comment:
You can eliminate the if statement since the `ifPresent` will already redundantly add this.